bindings/core/src/method_handler/wallet.rs

```diff
@@ -63,6 +63,7 @@ pub(crate) async fn call_wallet_method_internal(wallet: &Wallet, method: WalletM
             }
             Response::Accounts(account_dtos)
         }
+        #[cfg(feature = "storage")]
```

Comment:
Had a warning?

Reply:
Compilation error if one tried to compile without any features